### DISCUSSION TITLE
[@types/web-push] is missing a few options in RequestOptions

### DISCUSSION THREAD

**Discussion by NateLevin1** [DefinitelyTyped/DefinitelyTyped#66487] [Issues with a @types package] [answered]
[This PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53997) from 2 years ago added in some new RequestOptions, but 2 more have been added since that was merged.

Here are the docs for the two that need to be added:
- `urgency` is to indicate to the push service whether to send the notification immediately or prioritize the recipient’s device power considerations for delivery. Provide one of the following values: very-low, low, normal, or high. To attempt to deliver the notification immediately, specify high.
- `topic` optionally provide an identifier that the push service uses to coalesce notifications. Use a maximum of 32 characters from the URL or filename-safe Base64 characters sets.

I am ok with PRing this if needed. The solution is likely just to add the following to the `RequestOptions` interface on line 176 of `index.d.ts`:
```ts
urgency?: "very-low" | "low" | "normal" | "high" | undefined;
topic?: string | undefined;
```

[a comment by an automated account is not shown]

**Comment by paullessing** [comment, marked as answer]
@NateLevin1 Thanks for raising this! I've opened a PR to update the types: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66490
